postgres/datadog_checks/postgres/data/conf.yaml.example

```diff
@@ -265,12 +265,6 @@ instances:
         #
         # collection_interval: 10
 
-        ## @param pg_stat_statements_max_warning_threshold - number - optional - default: 10000
-        ## Set the threshold for a safe value of `pg_stat_statements.max`. If the database is configured with a value
-        ## higher than this threshold, a warning is emitted to recommend lowering the setting value.
-        #
-        # pg_stat_statements_max_warning_threshold: 10000
```

Comment:
The `hidden` field was in the wrong location for this option which resulted in creating this example.